pilot/pkg/model/jwks_resolver.go

```diff
@@ -247,19 +251,22 @@ func (r *JwksResolver) GetPublicKey(issuer string, jwksURI string) (string, erro
 		log.Errorf("Failed to jwks URI from %q: %v", issuer, err)
 	} else {
 		var resp []byte
-		resp, err = r.getRemoteContentWithRetry(jwksURI, networkFetchRetryCountOnMainFlow)
+		resp, err = r.getRemoteContentWithRetry(jwksURI, 0)
```

Comment:
This should be removed instead of just passing `0`.

Reply:
cool, thanks!!